### PR TITLE
homepage: added slider bar to control view position

### PR DESCRIPTION
**Pull request by AkifManzoor124**

### CODE REVIEW

File: Components/Home.js [hunk 1]
```diff
@@ -1,18 +1,69 @@
...
-    alignItems: 'center',
-    justifyContent: 'center',
+    backgroundColor: "grey",
+    // alignItems: 'center',
+    // justifyContent: 'center',
+  },
+  // circle: {
+  //   top: -50,
+  //   left: windowWidth-50,
+  //   top: -75,
```
Comment (AkifManzoor124) [inline Components/Home.js:55]: Probably a good idea to remove if unnecessary to have these comments

File: Components/Home.js [hunk 2]
```diff
@@ -1,18 +1,69 @@
...
+  //   height: 100,
+  //   borderRadius: 100 / 2,
+  //   backgroundColor: "green",
+  // },
+  circle: {
+    // top: -150 + {pos},
+    left: windowWidth/2,
+    width: 100,
+    height: 100,
+    borderRadius: 100 / 2,
```
Comment (AkifManzoor124) [inline Components/Home.js:66]: 50

File: Components/Home.js [hunk 3]
```diff
@@ -1,18 +1,69 @@
...
+        />
+        
+        <Text> {pos} </Text>
+        
+        {/* create a standing desk with shapes and move it with a slide bar */}
+
+      </View>
+
+      {/* <View style={styles.circle} /> */}
+      <View style={{top: -150 - pos*100, left: windowWidth/2, width: 100, height: 100, borderRadius: 100 / 2, backgroundColor: "pink"}} />
```
Comment (AkifManzoor124) [inline Components/Home.js:37]: Best to add the css name here instead of defining css values within component

File: Components/Home.js [hunk 4]
```diff
@@ -1,18 +1,69 @@
-import { StyleSheet, Text, View } from 'react-native';
+import { useState } from 'react';
+import { StyleSheet, Text, View, Button } from 'react-native';
+import { Dimensions } from 'react-native'
+
+import Slider from '@react-native-community/slider';
 
 export default function Home() {
+
+  const [pos,setPos] = useState(0);
```
Comment (AkifManzoor124) [inline Components/Home.js:9]: May be a good idea to define 0 as a constant

**Review by le-james — APPROVED** [review]

**Pull request merged**